hyprctl/main.cpp

```diff
@@ -341,6 +344,10 @@ int main(int argc, char** argv) {
         fullRequest += ARGS[i] + " ";
     }
 
+    if (quiet) {
+        std::cout.rdbuf(nullptr);
```

Comment:
I'd rather wrap std::cout in something like `log` that does `if (quiet) return;`